.github/workflows/ci.yml

```diff
@@ -142,6 +142,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
         with:
+          cache: false
```

Comment:
There is another cache handled by golangci-lint. This avoid errors when they both try to retrieve there contents

Reply:
We could have that in a comment in the yaml file.